pom.xml

```diff
@@ -37,9 +37,9 @@
     <inceptionYear>2017</inceptionYear>
 
     <properties>
-        <version.thorntail>2.2.1.Final</version.thorntail>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <version.thorntail>2.5.0.Final</version.thorntail>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Is this Java Version change a requirement or just a nice thing to have?

Reply:
@rdebusscher  It is required as I use the new Files API. Could be Java 9 though. Or I can refactor the code...  WDYT?

Reply:
I think Java11 is fine as this is our internal.

Reply:
@Karm No JDK11 is better than JDK9.